avaframe/log2Report/generateCompareReport.py

```diff
@@ -0,0 +1,247 @@
+"""
+    Generate a markdown report for data provided in dictionary with format to compare two simulations
+
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import glob
+import logging
+import numpy as np
+import shutil
+from avaframe.in3Utils import fileHandlerUtils as fU
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def fetchBenchParameters(avaDir):
+    """ Collect simulation parameter info from all standard tests """
+
+    # get name of avalanche
+    avaName = os.path.basename(avaDir)
+    avaDictName = avaName + 'Dict'
+
+    # set desired benchmark simulation info dictionary
+    if avaDictName == 'avaBowlDict':
+        avaDictName = {'simName': 'release1BL_null_dfa_0.155',
+                    'Simulation Parameters': {
+                        'type': 'list',
+                        'Release Area': 'releasehh1BL',
+                        'Entrainment Area': '',
+                        'Resistance Area': '',
+                        'Mu': '0.155',
+                        'Release thickness [m]': '1.000',
+                        'Release Mass [kg]': '20196.2',
+                        'Final Mass [kg]': '20196.2'},
+                    'Release area': {'type': 'columns', 'Release area scenario': 'release1BL'},
+                    'Test Info': {'type': 'text', 'Test Info': 'This test uses a bowl-shaped geometry.'}}
+
+    elif avaDictName == 'avaFlatPlaneDict':
+        avaDictName = {'simName': 'release1FP_null_dfa_0.155',
+                    'Simulation Parameters': {
+                        'type': 'list',
+                        'Release Area': 'release1FP',
+                        'Entrainment Area': '',
+                        'Resistance Area': '',
+                        'Mu': '0.155',
+                        'Release thickness [m]': '1.000',
+                        'Release Mass [kg]': '20000.',
+                        'Final Mass [kg]': '20000.'},
+                    'Release area': {'type': 'columns', 'Release area scenario': 'release1FP'},
+                    'Test Info': {'type': 'text', 'Test Info': 'This test runs on a flat plane geometry.'}}
```

Comment:
General: 
do we really all the  duplicated/unchanged/empty values? i.e. mu is the same , entr. and res are empty etc...

Reply:
It should include all the options that we can set, so that if there is a difference it is directly visible in the table, dont you think?